lib/pages/new_group/new_group_chat_info.dart

```diff
@@ -95,26 +92,16 @@ class NewGroupChatInfo extends StatelessWidget {
         child: ValueListenableBuilder<Either<Failure, Success>?>(
           valueListenable: newGroupController.createRoomStateNotifier,
           builder: (context, value, child) {
-            if (value == null) {
-              return child!;
-            } else {
-              return value.fold(
-                (failure) => child!,
-                (success) {
-                  if (success is CreateNewGroupChatLoading) {
-                    return const TwakeFloatingActionButton(
-                      customIcon: SizedBox(child: CircularProgressIndicator())
-                    );
-                  } else {
-                    return const SizedBox();
-                  }
-                });
+            if (value?.isRight() == true) {
```

Comment:
IMO, `isRight` is too specific. In the future, we maybe add some other temporary state inside creating room, maybe effect a lot for this button.

Reply:
moved to controller function 8e1f5767c7b099c2c9828ee8c810bb91e591429d